dask/dataframe/tests/test_groupby.py

```diff
@@ -404,6 +407,53 @@ def test_groupby_get_group():
         assert_eq(ddgrouped.a.get_group(2), pdgrouped.a.get_group(2))
 
 
+params = [
```

Comment:
Roughly how long to all these tests take? All else equal it's good to have more test coverage, but heavy parametrization can create very many tasks leading to slower collection / test times.

Reply:
On my machine locally, all of the 576 tests take around 40 seconds.